src/js/utils/ServicesPathUtil.js

```diff
@@ -0,0 +1,20 @@
+const ServicesPathUtil = {
```

Comment:
Let's please refactor the Breadcrumb component to use the feature rich Service (ServiceTree) interface instead of introducing an util to extract ids/paths from strings.